scripts/fuzz_opt.py

```diff
@@ -876,13 +876,13 @@ def randomize_opt_flags():
 
 After reduction, the reduced file will be in %(working_wasm)s
 ================================================================================
-                ''' % {'seed': seed,
-                       'original_wasm': original_wasm,
-                       'temp_wasm': os.path.abspath('t.wasm'),
-                       'working_wasm': os.path.abspath('w.wasm'),
-                       'wasm_reduce': in_bin('wasm-reduce'),
-                       'reduce_sh': os.path.abspath('reduce.sh')})
-                break
+            ''' % {'seed': seed,
+                   'original_wasm': original_wasm,
+                   'temp_wasm': os.path.abspath('t.wasm'),
+                   'working_wasm': os.path.abspath('w.wasm'),
+                   'wasm_reduce': in_bin('wasm-reduce'),
+                   'reduce_sh': os.path.abspath('reduce.sh')})
+            break
         if given_seed is not None:
```

Comment:
I think this `if` will now never be executed if a given seed fails? As the `break` above it always happens, taking us out of the loop which the `if` is inside. And specifically, we will never end up printing `finished running seed %d, see error above` (since we only reach that area if we pass).

How about changing this if to `if given_seed is not None: break` (if we get a seed, we must break after running it, whether or not it failed or succeeded), and move the `if given_seed_passed` to outside the loop, so it is always reached?

Reply:
Thanks for catching that! Done.

Reply:
Thanks for catching that. But that way if `given_seed` is `None` we don't break out of the loop even if we discovered a fuzzer error, no? I restored `break` here and extracted the `finished running seed ...` line at the end of the function. PTAL.

Reply:
Thanks for catching that. But should we take this `if` out of the loop to achieve that? I think adding `sys.exit(0)` in case `given_seed_passed` should be enough..? I also restored the deleted `break` above. PTAL.

Reply:
Thanks for catching that. Done. (I was confused and sent some other comments but they were incorrect so nvm)